backend/utils/file_storage/common_utils.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+
+class FileStorageUtil:
+    @staticmethod
+    def get_env_or_die(env_key: str) -> str:
```

Comment:
@harini-venkataraman please use any of the existing methods rather than redefining it again. I suggest using the [method from unstract-core](https://github.com/Zipstack/unstract/blob/30060abfc3bea2b640d98faf09b3420a83850dcf/unstract/core/src/unstract/core/utilities.py#L9-L28). Also the function `get_env_or_die` was named that way to indicate killing the tool container in case of SDK - let's name appropriately.

Let's remove this module altogether

Reply:
@chandrasekharan-zipstack This was added without the standard function and final decision on file structure for new files. It is removed after the changes are decided and moved